docs/medical-api/api-reference/patient/create-patient-external-sync.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "Create Patient External Sync"
+description: "Create a link between Metriport Patient and external system."
+api: "POST /medical/v1/patient/:id/external/sync"
+---
+
+Link the Metriport patient to an external EHR and synchronize their data.
```

Comment:
Suggesting to add more context, something like this?

> This endpoint can be used in situations where the Patient synchronization with external systems/EHR cannot be automatically performed, either because there's no capability on said system or because the Patient was created before the integration took place.
> 
> In these situations, customers are asked to create the Patient in Metriport and call this endpoint to synchronize the Patient with the EHR.